<launch>

   <node name="StateMachineNode" pkg="StateMachine" type="StateMachineNode" respawn="false" output="screen"> 
   </node> 

</launch>
